.github/workflows/release.yaml

```diff
@@ -49,5 +49,5 @@ jobs:
         uses: docker/build-push-action@v3
         with:
           push: true
-          tags: ghcr.io/${{env.REPOSITORY_OWNER_LOWER}}/experimental-binaries:${{github.ref_name}}
+          tags: ghcr.io/${{env.REPOSITORY_OWNER_LOWER}}/spring-experimental-binaries:${{github.ref_name}}
```

Comment:
Should this be `antelope-spring-experimental-binaries` to match `antelope-spring` name?

Reply:
I could go either way on this.. `antelope-spring` matches the names of the packages that are placed in here, but `spring` is the name of the repo.

I think maybe since we'll use this string along with other strings that are `antelope-spring` (the package name and the artifact name),
https://github.com/AntelopeIO/cdt/blob/9ecdca0f080b65e1339318cd2451a5f2593c4ee5/.github/workflows/build.yaml#L111-L115
it probably does make sense to make it `antelope-spring` just to keep everything looking similar in that block above. I'll update in a moment.